bundled/tool/lsp_zenml.py

```diff
@@ -261,6 +277,60 @@ def rename_stack(wrapper_instance, args):
         def copy_stack(wrapper_instance, args):
             """Copies a specified ZenML stack to a new stack."""
             return wrapper_instance.copy_stack(args)
+        
+        @self.command(f"{TOOL_MODULE_NAME}.createStack")
+        @self.zenml_command(wrapper_name="stacks_wrapper")
+        def create_stack(wrapper_instance, args):
+            """Creates a new ZenML stack."""
+            return wrapper_instance.create_stack(args)
```

Comment:
the word we use for this is register. would it be a lot of bother to not use the word create (either in code or in any error messages etc?

Reply:
Have changed all instances of create to register, code, errors, and comments.